portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -195,252 +195,254 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 			<liferay-ui:message arguments="<%= new Object[] {numberFormat.format(start + 1), numberFormat.format(end), numberFormat.format(total)} %>" key="showing-x-to-x-of-x-entries" />
 		</p>
 
-		<ul class="pagination">
-			<li class="page-item <%= (cur > 1) ? StringPool.BLANK : "disabled" %>">
-				<c:choose>
-					<c:when test="<%= cur > 1 %>">
-						<a class="lfr-portal-tooltip page-link" href="<%= _getHREF(formName, namespace + curParam, cur - 1, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, cur -1) : "" %>" title="<%= LanguageUtil.get(request, "previous-page") %>">
-					</c:when>
-					<c:otherwise>
-						<div class="page-link">
-					</c:otherwise>
-				</c:choose>
-
-					<liferay-ui:icon
-						icon='<%= PortalUtil.isRightToLeft(request) ? "angle-right" : "angle-left" %>'
-						markupView="lexicon"
-					/>
+		<nav aria-label="pagination">
+			<ul class="pagination">
+				<li class="page-item <%= (cur > 1) ? StringPool.BLANK : "disabled" %>">
+					<c:choose>
+						<c:when test="<%= cur > 1 %>">
+							<a class="lfr-portal-tooltip page-link" href="<%= _getHREF(formName, namespace + curParam, cur - 1, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, cur -1) : "" %>" title="<%= LanguageUtil.get(request, "previous-page") %>">
+						</c:when>
+						<c:otherwise>
+							<div class="page-link">
+						</c:otherwise>
+					</c:choose>
+
+						<liferay-ui:icon
+							icon='<%= PortalUtil.isRightToLeft(request) ? "angle-right" : "angle-left" %>'
+							markupView="lexicon"
+						/>
```

Comment:
CI SF fails on this line. To run full CI SF test locally, you can do:
- go to `/portal/portal-impl`
- run `ant format-source-current-branch`

This is not usually needed, in a standard OSGi module you can run `gradle formatSource`, but this covers odd places in DXP like `portal-web`.

If you run it, you'll see the error:
```
format-source-files:
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: Use <clay:icon> tag instead of <liferay-ui:icon>: /home/marko/git/lp-master/portal/portal-web/docroot/html/taglib/ui/page_iterator/start.jsp (SourceCheck:IllegalTaglibsCheck)

BUILD FAILED
```

We're doing this as a way to gradually fix tech debt. Replacing all `<liferay-ui:icon>`s would be extremely expensive, so we're only doing this on new commits. You will need to do this migration, to `<clay:icon>`. It is not strictly related to the PR task, but we must do it to satisfy SF.

Reply:
@markocikos Thanks for the review, as for the migration, I tried before to do that and move `<liferay-ui:icon>` to `<clay:icon>` but from what I've asked, it is not possible in `portal-web` to use `<clay:icon>` there, do you have any idea how to do that?

Reply:
@bryceosterhaus @izaera @dsanz Any ideas how to entangle this? Looks to me like we need to fix this in formatter. Who is responsible for this formatter?

Reply:
Probably we'll need some exception in the formatter, like [we already do](https://github.com/liferay/liferay-portal/blob/57b0eaf5cf0f9d8afb939948710d79c8b10d95ea/portal-web/source-formatter-suppressions.xml#L5). @smailbiala pls add an extra commit with an explanation similar to [this](https://github.com/liferay/liferay-portal/commit/57b0eaf5cf0f9d8afb939948710d79c8b10d95ea)
thx